wpi-many.sh

```diff
@@ -0,0 +1,217 @@
+#!/bin/bash
+
+# This script runs the Checker Framework's whole-program inference on each of a list of projects.
+
+### Usage
+
+# - Clone the wpi-many repository containing this script on the experimental
```

Comment:
The usage instructions are duplicated, and are slightly different, here and in the README file.  To avoid duplication and confusion, give the instructions in just one of the two places.  The other place can just point to the one with the instructions.